components/application-broker/Dockerfile

```diff
@@ -6,16 +6,17 @@ ADD . $SRC_DIR
 WORKDIR $SRC_DIR
 
 RUN CGO_ENABLED=0 go build -o /application-broker ./cmd/broker
+RUN apk add --no-cache curl
```

Comment:
why u need that? it's not copied to the final image

Reply:
removed